packages/sync/src/offline/CordovaNetworkStatus.ts

```diff
@@ -19,6 +15,6 @@ export class CordovaNetworkStatus implements NetworkStatus {
   }
 
   public isOffline(): boolean {
-    return (navigator.connection.type === this.NONE);
```

Comment:
I recognize this change, but I would love to test this carefully and see how this will interact with deviceReady